lib/cached_network_image.dart

```diff
@@ -170,11 +270,11 @@ class CacheManager {
     var response;
     try {
       response = await http.get(url, headers: headers);
-    }catch(e){}
+    } catch (e) {}
     if (response != null) {
       if (response.statusCode == 200) {
         await newCache.setDataFromHeaders(response.headers);
-        var folder =  new File(newCache.filePath).parent;
+        var folder = new File(newCache.filePath).parent;
         if (!(await folder.exists())) {
           folder.createSync(recursive: true);
         }
```

Comment:
looks like sole folders are being created. I am not sure if we delete any of these folders..
what is the reason to have these folders? I thought that files will have just GUIDS as names so could stored flat?

Reply:
Yes they are practically stored flat. It only creates one folder so all cache files are stored in a folder and cannot conflict with random other files in de cache folder.